src/libs/registerForum.ts

```diff
@@ -15,7 +15,8 @@ const registerForum = (
   subscriptions: ExtensionContext["subscriptions"],
   forumProvider: ForumTitleProvider,
   cookieJar: CookieJar,
-  threadProvider: ThreadProvider
+  threadProvider: ThreadProvider,
+  currentThread: ThreadTitle | null = null
 ) => {
   subscriptions.push(
```

Comment:
currentThread 作为变量放在函数里吧，不用放参数上...
```suggestion
  threadProvider: ThreadProvider
) => {
  let currentThread: ThreadTitle | null = null;
  subscriptions.push(
```